### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @Nofims

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
```
Comment (SergeyTeslov) [inline Program.cs:1]: что это за файл?

File: Laba2/Laba2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+
+                Console.Write("Ввелите число х : ");
+            double x = double.Parse(Console.ReadLine());
+
```
Comment (SergeyTeslov) [inline Laba2/Laba2/Program.cs:18]: где вывод значений через цикл ? от [-9,7] с шагом 0.2?

Comment (SergeyTeslov) [reply Laba2/Laba2/Program.cs:18]: image
не очевидно что выводит консоль когда вводится переменная x
консоль сворачивается после выполнения , приходится запускать через debutg (ctrl + f5)

**Comment by SergeyTeslov** [comment]
Оценка: 2/5

**Comment by Nofims** [comment]
И почему не очевидно , если там функция равна нулю

**Pull request merged**